__TESTS__/unit/values/format/Foramt.test.ts

```diff
@@ -0,0 +1,83 @@
+import {Format} from "../../../../src/values/format/Format";
+
+describe('Test imported values', () => {
+  it ('Tests audio formats', () => {
+
+    const {audioAac, audioAiff, audioAmr, audioFlac, audioM4a, audioMp3, audioOgg, audioOpus, audioWav} = Format;
+    expect(audioAac().toString()).toBe('aac');
+    expect(audioAiff().toString()).toBe('aiff');
+    expect(audioAmr().toString()).toBe('amr');
+    expect(audioFlac().toString()).toBe('flac');
+    expect(audioM4a().toString()).toBe('m4a');
+    expect(audioMp3().toString()).toBe('mp3');
+    expect(audioOgg().toString()).toBe('ogg');
+    expect(audioOpus().toString()).toBe('opus');
+    expect(audioWav().toString()).toBe('wav');
+  });
+
+  it('Test Video formats', () => {
+    const {video3g2, video3gp, videoAvi, videoFlv, videoM2ts, videoM3u8, videoMkv, videoMov, videoMp4, videoMpd,
+      videoMpeg, videoMts, videoMxf, videoOgv, videoTs, videoWebm, videoWmv} = Format;
+
+    expect(video3g2().toString()).toBe('3g2');
```

Comment:
You think maybe it should be all upper case for the extensions? videoMKV instead of videoMkv?

Reply:
Part of the spec, we can raise it there if you want, but for now, this is how it was decided.